mobile/lib/main.dart

```diff
@@ -275,9 +275,15 @@ Future<void> _scheduleHeartBeat(
 }
 
 Future<void> _scheduleFGHomeWidgetSync() async {
```

Comment:
What's the reason for scheduling it after 15 minutes?

Reply:
On login two services will try to populate home, one is this and other is favorite service initFav. So I had to resolve this conflict by delaying this.